setup.py

```diff
@@ -25,7 +25,7 @@ def read(*names, **kwargs):
 
 setup(
     name='pytest-cython',
-    version='0.1.0',
+    version='0.1.1',
     license='MIT',
     description='A plugin for testing Cython extension modules',
     long_description='%s\n%s' % (
```

Comment:
I forgot, `setup.py` should also add `long_description_content_type='text/x-rst'` now.